whitepaper/whitepaper.md

```diff
@@ -886,207 +886,116 @@ APIs of Verifier and Validator are described as the following table:
 
 | No. | Component | API Name | Input | Description |
 | --- | --- | --- | --- | --- |
-| 1. | Verifier | getVerifierInformation | none | Get the verifier information including version, name, ID, and other information |
-| 2. | Verifier | getSmartContractList | none | Get the list of available smart contracts at the connected ledger |
-| 3. | Verifier | sendSignedTransaction | `signedTransaction`(string) | Request a verifier to execute a ledger operation |
-| 4. | Verifier | getBalance | `address`(string) | Get balance of an account for native token on a ledger |
-| 5. | Verifier | execSyncFunction | `address`(string)<br>`funcName`(string)<br>`args`(string[]) | Execute a synchronous function held by a smart contract |
-| 6. | Verifier | startMonitor | `clientId`(string)<br>`cb`(function) | Request a verifier to start monitoring ledger |
-| 7. | Verifier | stopMonitor | `clientId`(string) | Rrequest a verifier to stop monitoring ledger |
-| 8. | Verifier | connect | `validatorURL`(string)<br>authentication credential | request a validator to start a bi-directional communication channel via a verifier |
-| 9. | Verifier | disconnect | none | request a validator to stop a bi-directional communication channel via a verifier |
-| 10. | Validator | getValidatorInformation | `validatorURL`(string) | Get the validator information including version, name, ID, and other information |
-| 11. | Verifier | getSmartContractList | none | Get the list of available smart contracts at the connected ledger |
-| 12. | Validator | sendSignedTransaction | `signedTransaction`(string) | Send already-signed transactions to a ledger |
-| 13. | Validator | getBalance | `address`(string) | Get balance of an account for native token on a ledger |
-| 14. | Validator | execSyncFunction | `address`(string)<br>`funcName`(string)<br>`args`(string[]) | Execute a synchronous function held by a smart contract |
-| 15. | Validator | startMonitor | `clientId`(string)<br>`cb`(function) | Request a validator to start monitoring ledger |
-| 16. | Validator | stopMonitor | `clientId`(string) | Request a validator to stop monitoring ledger |
+| 1. | Verifier | sendAsyncRequest | `contract` (object)<br>`method` (object)<br>`args` (object) | Sends an asynchronous request to the validator |
```

Comment:
@yayoiukai2021 Were you intended to change the whitepaper.md or was that by accident?